packages/tocco-ui/src/FormattedValue/typeFormatters/MultiSelectFormatter.js

```diff
@@ -6,14 +6,17 @@ import Typography from '../../Typography'
 export const MultiSeparator = () => ', '
 
 const MultiSelectFormatter = ({value, options, breakWords}) => {
+  const {navigationStrategy, linkProps} = options || {}
   return (
     <Typography.Span breakWords={breakWords}>
       {value && value.length > 0
```

Comment:
Can we put the resulting elements into a seperate var? I think it would be cleaner...
Can we shorten this query? From `value && value.length > 0` to `value?.length > 0`